### ISSUE TITLE
bandPassFilter should fail if lowCutOff>highCutOff?

### ISSUE THREAD

**Issue by MartinSchmitzDo** [psambit9791/jDSP#7]
If you run any Frequency filter and have lowCutOff>highCutOff, then you do not get an error. In fact this happens:

```
double centreFreq = (highCutoff + lowCutoff)/2.0;
double width = Math.abs(highCutoff - lowCutoff);
```

So what happens here is lowCutOff and highCutOff are implicitly inverted. I think this is not desired?

**Comment by psambit9791** [comment]
Right. That is true.

Will take a look it at.

**Comment by psambit9791** [comment]
The issue has been fixed with commit [98d55f9691ca365963de219f9390725e9b3375fd](https://github.com/psambit9791/jDSP/commit/98d55f9691ca365963de219f9390725e9b3375fd).
